CommandsManual.md

```diff
@@ -1655,6 +1683,7 @@ At the end of the command execution, information about the test configuration is
 | --test-file | -f | No | Yes | Test file name or pattern. All files located in the current directory and all its sub-directories whose names match the specified name or pattern are considered as files with test cases. This option may be repeated multiple times to specify multiple names and/or patterns. The values of the repeated option are combined by logical OR. Default: `"*.test.nut"` `"tests/**/*.test.nut"` |
 | --github-config | -i | No | Yes | A path to a GitHub credentials file. A relative or absolute path can be used. The specified file may not exist |
 | --bitbucket-srv-config | -b | No | Yes | A path to a Bitbucket Server credentials file. A relative or absolute path can be used. The specified file may not exist |
+| --azure-repos-config | -az | No | Yes | A path to an Azure Repos credentials file. A relative or absolute path can be used. The specified file may not exist |
```

Comment:
It's better to use single letter alias if possible.
For example `-r` looks like a good candidate. Please check if there are any conflicts with it.